src/sass/file-preview-card/stories/file-preview-card.stories.js

```diff
@@ -0,0 +1,264 @@
+export default { title: "Skin/File Preview Card" };
+
+export const uploading = () => `
+    <div style="height: 168px; width: 168px;">
+        <div class="file-preview-card">
+            <div class="file-preview-card__center">    
+                <span class="progress-spinner progress-spinner--large" aria-label="Busy" role="img">
+                    <svg class="icon icon--30" aria-hidden="true">
+                        <use href="#icon-spinner-30"></use>
+                    </svg>
+                </span>
+            </div>
+            <div class="file-preview-card__menu-action">
+                <button class="icon-btn" type="button" aria-label="Cancel upload">
+                    <svg class="icon icon--16" height="16" width="16" aria-hidden="true">
+                        <use href="#icon-close-16"></use>
+                    </svg>
+                </button>
+            </div>
+        </div>
+    </div>
+`;
+
+export const uploadError = () => `
+    <div style="height: 168px; width: 168px;">
+        <div class="file-preview-card">
+            <div class="file-preview-card__center">    
+                <div class="inline-notice inline-notice--attention">
+                    <span class="inline-notice__header">
+                        <svg class="icon--attention-filled icon icon--16" aria-hidden="true">
+                            <use href="#icon-attention-filled-16"></use>
+                        </svg>
+                    </span>
+                    <span class="inline-notice__main">
+                        <p>Upload failed</p>
```

Comment:
Doesn't this individual card need something to indicate an error for screen readers?

Reply:
This is removed